src/components/CombinationContainer/index.js

```diff
@@ -32,12 +32,13 @@ export default function CombinationContainer() {
   return (
     <div className="CombinationContainer">
       <Select
-        // eslint-disable-next-line no-shadow
-        onChange={(sortingOptionIndex) => patchTermData({ sortingOptionIndex })}
         value={sortingOptionIndex}
         options={oscar.sortingOptions.map((sortingOption, i) => ({
-          value: i,
-          label: sortingOption.label
+          innerValue: i,
+          innerLabel: sortingOption.label,
+          // eslint-disable-next-line no-shadow
+          onClick: (sortingOptionIndex) =>
```

Comment:
Is there a reason the onChange became an onClick here?

Reply:
Yes, for consistency reason, I changed all select to change its value based on onClick() function. The general <Select /> now uses onClick = {() => onClick(value)} as opposed to the original onClick = {() => onChange(value)}.